src/QMCDrivers/QMCDriverFactory.cpp

```diff
@@ -92,7 +93,11 @@ QMCDriverFactory::DriverAssemblyState QMCDriverFactory::readSection(int curSerie
   if (curName != "qmc")
     qmc_mode = curName;
   int nchars = qmc_mode.size();
-  if (qmc_mode.find("linear") < nchars)
+  if (qmc_mode.find("linear_batch") < nchars)
+  {
+    das.new_run_type = QMCRunType::LINEAR_OPTIMIZE_BATCH;
```

Comment:
I mean the enum is used to label drivers here. If we don't need "opt/opt_batch" but only "linear/linear_batch", no need to connect "opt/opt_batch". Anyway, should not block this PR.